source/delegate/README.md

```diff
@@ -38,3 +38,5 @@ Run an instance of `ganache-cli` before running tests.
 ```
 yarn ganache
 ```
+
+Delegate tests require build artifacts from `@airswap/swap` and `@airswap/tokens`. Before running tests, run `yarn compile` in both the [swap](../swap) and [tokens](../tokens) packages in this repository.
```

Comment:
While this is true. Why not instruct them to run `yarn compile` from the root directory?

Reply:
Yep turns out `order-utils` depends on builds so everything should be compiled prior to running tests on an individual package.